code/modules/mob/living/simple_animal/hostile/jungle_mobs.dm

```diff
@@ -353,4 +360,215 @@
 			return
 	icon_state = "leaper"
 
-#undef PLAYER_HOP_DELAY
\ No newline at end of file
+#undef PLAYER_HOP_DELAY
+
+////JUNGLE MOOK////
+
+#define MOOK_ATTACK_NEUTRAL 0
+#define MOOK_ATTACK_WARMUP 1
+#define MOOK_ATTACK_ACTIVE 2
+#define MOOK_ATTACK_RECOVERY 3
+
+#define ATTACK_INTERMISSION_TIME 5
+
+/mob/living/simple_animal/hostile/jungle/mook
+	name = "wanderer"
+	desc = "This unhealthy looking primitive is wielding a rudimentary hatchet, swinging it with wild abandon. One isn't much of a threat, but in numbers they can quickly overwhelm a superior opponent."
+	maxHealth = 45
+	health = 45
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	icon = 'icons/mob/jungle/arachnid.dmi'
+	icon_state = "mook"
+	icon_living = "mook"
+	icon_dead = "mook_dead"
+	pixel_x = -16
+	pixel_y = -16
```

Comment:
almost (actually no?) no mobs/objects extend below their turf.
Plenty extend NORTH, SOUTH and EAST out of their turf, but none below (SOUTH).